cmd/osbuild-composer/config.go

```diff
@@ -14,7 +15,7 @@ type ComposerConfigFile struct {
 	Koji          KojiAPIConfig     `toml:"koji"`
 	Worker        WorkerAPIConfig   `toml:"worker"`
 	WeldrAPI      WeldrAPIConfig    `toml:"weldr_api"`
-	DistroAliases map[string]string `toml:"distro_aliases"`
+	DistroAliases map[string]string `toml:"distro_aliases" env:"DISTRO_ALIASES"`
```

Comment:
This should have an `OSBUILD_` prefix. Just in case.

Reply:
I considered it, but then saw that the other configuration options also do not use any prefix. :shrug:

Reply:
Odd. When I added `OSBUILD_LOCALSAVE` that was one of the changes folks requested. In practice it likely doesn't really matter.